drud/ddev
# Allow '1' and '2' settings for composer_version

## The Problem/Issue/Bug:

People will want to use either latest v1 or latest v2 of composer, or perhaps a specific version.

This PR lets composer_version be "1", "2", a specific version, or "" (which should get latest v1 presently)

## How this PR Solves The Problem:

## Manual Testing Instructions:

`ddev exec composer --version` should get latest v1
`ddev config --composer-version=1 && ddev start && ddev exec composer --version` should get latest v1
`ddev config --composer-version=2 && ddev start && ddev exec composer --version` should get latest v2


## Automated Testing Overview:

Added TestComposerVersion

## Release/Deployment notes:

Explain how this works in release notes. It's slightly different from in alpha8 ("latest" is gone)
